pages/packages/[package]/versions/[version]/topics/[topic].tsx

```diff
@@ -130,8 +133,32 @@ export default function TopicPage({ topicData }: Props) {
           )}
           {examples && (
             <section>
-              <h2>Examples</h2>
-              <pre>{examples}</pre>
+              <div className="relative">
+                <h2>Examples</h2>
+                <a
+                  className="absolute p-2 text-sm rounded-md top-0	right-0 hover:bg-green-400 md:p-3 md:text-base md:top-16 md:right-2.5"
+                  href={`https://www.datacamp.com/workspace?utm_source=r-docs&utm_medium=docs&utm_term=${topic}&utm_content=run_example_in_workspace`}
```

Comment:
Why not send people straight to a workspace using the /new endpoint, starting from the base-r template that is almost entirely empty?